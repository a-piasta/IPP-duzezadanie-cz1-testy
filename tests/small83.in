addRoad;peqehvuezmbpxiefdzxjibio;wrgjhjylzfvwzzwncn;15;2
removeRoad;peqehvuezmbpxiefdzxjibio;wrgjhjylzfvwzzwncn
newRoute;145;vcqwplhngsuqejdxehobwlq;yncdycbixotpzr
addRoad;lrvepvmxpeblondc;yncdycbixotpzr;7;13
addRoad;wrgjhjylzfvwzzwncn;hbxdvbwqgmynqsi;8;2
newRoute;645;peqehvuezmbpxiefdzxjibio;yncdycbixotpzr
getRouteDescription;645
removeRoad;yncdycbixotpzr;lrvepvmxpeblondc
newRoute;653;fqclayhsljqlycefddjaehqmtoclg;wrgjhjylzfvwzzwncn
repairRoad;vcqwplhngsuqejdxehobwlq;wrgjhjylzfvwzzwncn;-4
addRoad;vcqwplhngsuqejdxehobwlq;peqehvuezmbpxiefdzxjibio;1;14
extendRoute;145;yncdycbixotpzr
repairRoad;fqclayhsljqlycefddjaehqmtoclg;peqehvuezmbpxiefdzxjibio;-12
getRouteDescription;145
removeRoad;hbxdvbwqgmynqsi;wrgjhjylzfvwzzwncn
repairRoad;vcqwplhngsuqejdxehobwlq;dywhjynqocmcexkqyasbaxc;-11
getRouteDescription;645
newRoute;589;lrvepvmxpeblondc;dywhjynqocmcexkqyasbaxc
repairRoad;wrgjhjylzfvwzzwncn;lrvepvmxpeblondc;6
addRoad;wrgjhjylzfvwzzwncn;ifbusvmipejuwqmbxynm;14;-12
addRoad;fqclayhsljqlycefddjaehqmtoclg;wrgjhjylzfvwzzwncn;10;12
extendRoute;589;ifbusvmipejuwqmbxynm
newRoute;507;lrsqspzcshsdfjlotungwubstd;lrsqspzcshsdfjlotungwubstd
newRoute;699;yncdycbixotpzr;ifbusvmipejuwqmbxynm
addRoad;peqehvuezmbpxiefdzxjibio;lrsqspzcshsdfjlotungwubstd;17;-6
newRoute;709;lrvepvmxpeblondc;lrsqspzcshsdfjlotungwubstd
repairRoad;dywhjynqocmcexkqyasbaxc;vcqwplhngsuqejdxehobwlq;-15
removeRoad;wrgjhjylzfvwzzwncn;fqclayhsljqlycefddjaehqmtoclg
repairRoad;lrvepvmxpeblondc;peqehvuezmbpxiefdzxjibio;-11
repairRoad;vcqwplhngsuqejdxehobwlq;ifbusvmipejuwqmbxynm;-1
extendRoute;507;vcqwplhngsuqejdxehobwlq
newRoute;618;fqclayhsljqlycefddjaehqmtoclg;yncdycbixotpzr
extendRoute;709;lrsqspzcshsdfjlotungwubstd
removeRoad;peqehvuezmbpxiefdzxjibio;wrgjhjylzfvwzzwncn
addRoad;lrsqspzcshsdfjlotungwubstd;lrvepvmxpeblondc;13;-7
addRoad;wrgjhjylzfvwzzwncn;wrgjhjylzfvwzzwncn;7;7
extendRoute;507;hbxdvbwqgmynqsi
getRouteDescription;589
extendRoute;645;ifbusvmipejuwqmbxynm
newRoute;698;fqclayhsljqlycefddjaehqmtoclg;wrgjhjylzfvwzzwncn
extendRoute;709;lrvepvmxpeblondc
getRouteDescription;709
removeRoad;wrgjhjylzfvwzzwncn;ifbusvmipejuwqmbxynm
addRoad;hbxdvbwqgmynqsi;ifbusvmipejuwqmbxynm;7;2
newRoute;149;hbxdvbwqgmynqsi;vcqwplhngsuqejdxehobwlq
newRoute;309;ifbusvmipejuwqmbxynm;fqclayhsljqlycefddjaehqmtoclg
addRoad;peqehvuezmbpxiefdzxjibio;lrvepvmxpeblondc;27;-3
repairRoad;peqehvuezmbpxiefdzxjibio;dywhjynqocmcexkqyasbaxc;5
newRoute;49;dywhjynqocmcexkqyasbaxc;ifbusvmipejuwqmbxynm
getRouteDescription;149
repairRoad;vcqwplhngsuqejdxehobwlq;fqclayhsljqlycefddjaehqmtoclg;14
newRoute;778;hbxdvbwqgmynqsi;hbxdvbwqgmynqsi
extendRoute;618;peqehvuezmbpxiefdzxjibio
newRoute;786;lrvepvmxpeblondc;hbxdvbwqgmynqsi
newRoute;223;yncdycbixotpzr;peqehvuezmbpxiefdzxjibio
removeRoad;peqehvuezmbpxiefdzxjibio;lrvepvmxpeblondc
getRouteDescription;856
extendRoute;778;ifbusvmipejuwqmbxynm
newRoute;202;vcqwplhngsuqejdxehobwlq;dywhjynqocmcexkqyasbaxc
