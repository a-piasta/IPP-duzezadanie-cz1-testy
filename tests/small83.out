1: TAK
2: TAK
3: NIE
4: TAK
5: TAK
6: NIE
7: 
8: TAK
9: NIE
10: NIE
11: TAK
12: NIE
13: NIE
14: 
15: TAK
16: NIE
17: 
18: NIE
19: NIE
20: TAK
21: TAK
22: NIE
23: NIE
24: NIE
25: TAK
26: NIE
27: NIE
28: TAK
29: NIE
30: NIE
31: NIE
32: NIE
33: NIE
34: NIE
35: TAK
36: NIE
37: NIE
38: 
39: NIE
40: NIE
41: NIE
42: 
43: TAK
44: TAK
45: NIE
46: NIE
47: TAK
48: NIE
49: NIE
50: 
51: NIE
52: NIE
53: NIE
54: NIE
55: NIE
56: TAK
57: 
58: NIE
59: NIE
